src/history/__tests__/historyActions.test.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  CHANGE_DATE_FILTER_ACTION,
+  CHANGE_PRICE_FILTER_ACTION,
+  changeDateFilterAction,
+  changePriceFilterAction,
+  getHistoryByDateFilter,
+  getHistoryByPriceFilter,
+  getHistoryData,
+  HISTORY_RECEIVED_ACTION,
+  historyReceivedAction
+} from 'history/historyActions'
+import configureMockStore from 'redux-mock-store'
+import thunk from 'redux-thunk'
+
+import { mockChartData, mockEmptyHistoryState, mockHistoryState } from './mockData'
+
+const mockApi = {
```

Comment:
You can perhaps move this out to __mock__/api.mock.ts and import the module. Separation of Concern